src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -393,7 +393,7 @@ public static string CombineAbsoluteAndRelativePaths(string root, string relativ
         {
             Debug.Assert(IsAbsolute(root));
 
-            return CombinePossiblyRelativeAndRelativePaths(root, relativePath);
+            return CombinePossiblyRelativeAndRelativePaths(root, relativePath)!;
```

Comment:
I don't see how this is safe?

Reply:
I'm not sure what I was thinking. Will remove. Thanks

---
In reply to: [377806395](https://github.com/dotnet/roslyn/pull/41063#discussion_r377806395) [](ancestors = 377806395)